airflow/Dockerfile

```diff
@@ -13,9 +13,6 @@ ARG STATSD_EXPORTER
 ARG TARGETARCH
 ARG TARGETOS
 
-# https://github.com/hadolint/hadolint/wiki/DL4006
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-
```

Comment:
**question**: why did the shell get unset in some images?

Reply:
It should be unset in all images actually.

I can revert this but it was inconsistent between images and I looked and it seems as if it is inherited by the base images so we only have to define this once.

I don't feel strongly about this though. I can also leave it in and just make it consistent.